### ISSUE TITLE
Guide: don't use GlobalScope in first example

### ISSUE THREAD

**Issue by zach-klippenstein** [Kotlin/kotlinx.coroutines#2122]
The use of `GlobalScope` is well-known to be a code smell in most cases - it's documented in the kdoc itself, as well as numerous articles and discussions around the Kotlin community. 

However, in the official coroutines guide, the very first example of coroutines code, and the first example of such code most people will probably see from an "official" source, uses `GlobalScope`. I think this is counterproductive to all the other recommendations and warnings, and sets many people's first impressions of it as something that _is_ ok to use as a default, and not only as a last resort. 

I think the guide would do a better job of getting people off to the right start if the early examples, at least, didn't use `GlobalScope`.

**Comment by elizarov** [comment]
Good point. It is not a simple fix, though. It requires to rethink the whole approach to the introduction of coroutines.
